scalecube-configuration-api/src/main/java/io/scalecube/configuration/api/ReadEntryHistoryRequest.java

```diff
@@ -12,31 +12,21 @@
    *
    * @deprecated only for serialization/deserialization.
    */
-  FetchRequest() {}
+  ReadEntryHistoryRequest() {}
 
   /**
-   * Constructs a FetchRequest object.
+   * Constructs a ReadEntryRequest object.
```

Comment:
Since this class is `ReadEntryHistoryRequest`, so java doc should describe the same class name.